plugins/gradle/gradle-plugin/pom.xml

```diff
@@ -31,6 +31,12 @@
         </configuration>
       </plugin>
     </plugins>
+    <resources>
+      <resource>
+        <directory>${project.basedir}/src/main/resources</directory>
+        <filtering>true</filtering>
+      </resource>
+    </resources>
```

Comment:
Filtering the resources for embedding the Thorntail project version in to the properties file.